.github/workflows/browserstack-prepare-artifacts.yaml

```diff
@@ -26,7 +26,7 @@ on:
         required: true
 jobs:
   prepare-device-farm-artifacts:
-    runs-on: macos-latest
+    runs-on: ubuntu-latest
```

Comment:
Running builds on `ubuntu-latest` is 2x+ faster for Android builds than `macos-latest`...